include/psuedocode.md

```diff
@@ -0,0 +1,115 @@
+The follow outlines some psuedocode for how to implement the logic for a prototype QEO solution.
+
+```
```

Comment:
"```c" should be able to visualize as C

Reply:
But it's not really C. I use a mix of stuff. Let me see if marking a particular language helps.